test/upgrade/continual/channel-config.toml

```diff
@@ -0,0 +1,7 @@
+# logLevel = 'DEBUG'
+[sender]
+address = '{{- .Endpoint -}}'
+interval = {{ .Config.Interval.Nanoseconds }}
```

Comment:
why is the last char here _red_ ?

Reply:
This is because this is template of a toml, but file is names `*.toml`. Github thinks this is regular TOML, and then it's invalid.

The `interval = {{ .Config.Interval.Nanoseconds }}` is replaced and actual TOML file is something like: `interval = 10000000`, and that's valid TOML.